unlink-finder/main.js

```diff
@@ -46,6 +46,7 @@ function getAllAliases() {
     uid: p.uid,
     aliases:
       getConfigFromPage(p.title)
+        ?.Aliases.replace(/(^\s*,)|(,\s*$)/g, '')
         ?.Aliases?.split(",")
         ?.map((a) => a.trim()) || [],
```

Comment:
the `replace` will return a string, which won't have an `Aliases` field on the next line which will return no aliases

I think we could achieve the same effect if we add a `?.filter((a) => !!a)` after the `map`, to filter for non empty references